build.gradle

```diff
@@ -74,14 +78,39 @@ runIde {
     jvmArgs '--add-exports', 'java.base/jdk.internal.vm=ALL-UNNAMED'
 }
 
+
+task buildJakartaCore(type: Exec) {
```

Comment:
My understanding is lsp4jakarta is a separate project. Do we need these build instructions here?

Reply:
Our ls-integration wants to include lsp4jakarta, which I believe is not published anywhere yet, so we have to clone and build/install it locally. I figured `build.gradle` was the most convenient place to put it for developing, should it be somewhere else?

Reply:
There is a build script for lsp4jakarta in the root directory of that project called `buildAll.sh`. The code in liberty-tools-intellij should just be for building itself. The interface to other projects is contained in the `dependencies {}` section. 

If one wanted to build both lsp4jakarta and liberty-tools-intellij in one go perhaps a local script in the directory containing these two projects.

Reply:
Now that LSP4Jakarta has snapshots available, we should pull from the Eclipse repo. We will address this with future changes.

Reply:
Will review in future.